topo/sparse/sparse.pyx

```diff
@@ -11,7 +11,7 @@ the sparse matrix are available.
 
 from cython.operator cimport dereference as deref
 from cpython cimport bool
-import numpy
+import numpy, scipy
 cimport numpy
```

Comment:
Sorry about not replying. Currently the SciPy is used for transforming the sparse projection data into CSR format, before it's transferred to the GPU. There is a way to avoid using SciPy's sparse matrices though, which I'll try to implement.